clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -895,6 +899,10 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
     if (channelOptionWriteBufferHighBytes <= 0) {
       throw new VeniceException("Invalid channel option write buffer high bytes: " + channelOptionWriteBufferHighBytes);
     }
+    aaWCWorkloadParallelProcessingEnabled =
+        serverProperties.getBoolean(SERVER_AA_WC_WORKLOAD_PARALLEL_PROCESSING_ENABLED, false);
+    aaWCWorkloadParallelProcessingThreadPoolSize =
+        serverProperties.getInt(SERVER_AA_WC_WORKLOAD_PARALLEL_PROCESSING_THREAD_POOL_SIZE, 8);
```

Comment:
Actually I have a question - I think correctness there is no issue, but if all the AAWC leader can submit batch of record pulled from RT, then would this be true for current / non-current version to share the same critical path and one maybe impact the other?

Reply:
Yes, it is correct.
For the current/non-current version differentiation, we will delegate it to the consumer pool strategy, rather than solving it here.

There are different layer of strategies, and this layer is purely for scheduling, not prioritization.
That is the way how it is designed originally: separate of concerns.

Reply:
Synced offline, I think for now it is not a concern as described above.